templates/stamp/stamp.parameters.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterName": {
+            "value": "loxabvtwuc1"
+        },
+        "clusterNameDomainName": {
+            "value": "oxadev"
+        },
+        "apiVersion": {
+            "value": "2016-09-01"
+        },
+        "osFamilySpec": {
+            "value": "ubuntu"
+        },
+        "jumpBoxVmSize": {
+            "value": "Standard_DS1_v2"
+        },
+        "storageApiVersion": {
+            "value": "2016-01-01"
+        },
+        "adminUsername": {
+            "value": "lexoxaadmin"
+        },
+        "adminPublicKey": {
+            "value": "stringvalue"
+        },
+        "backendAdminPassword": {
+            "value": "securestringvalue"
+
+        }
```

Comment:
Actually, this file isn't even needed. There already is an environment-based parameter.json file intended for the private repo